packages/adapters/wagmi/src/client.ts

```diff
@@ -253,6 +253,34 @@ export class WagmiAdapter extends AdapterBlueprint {
       }
     }
 
+    // Check if the app is running inside an iframe and the referrer is from Safe
```

Comment:
Adding third party connectors are getting bigger, maybe we could utilize these third party setups?